example/src/Main.hs

```diff
@@ -12,13 +12,13 @@ import GHC.Generics
 
 data AppConfig = AppConfig
   { appConfigServer :: Settings
-  , appConfigSeed :: Int
+  , _appConfig_seed :: Int
   } deriving (Generic)
 instance Conferer.FromConfig AppConfig
 instance Conferer.DefaultConfig AppConfig where
   configDef = AppConfig
     { appConfigServer = setPort 2222 Conferer.configDef -- If you want to configure new default for internal libs this is the place
-    , appConfigSeed = 17
+    , _appConfig_seed = 17
```

Comment:
try to avoid changing files that are not directly related with what you want to change

Reply:
I reverted, but I didn't get the issue. This is example project. Its goal to represent library features. I added new feature and showed how to use it. There are several field using original convention. I guess 1 field per notation is enough. Plus it helped me to test.